config/argocd/kubernetes/argocd-cm.yaml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Shall we use `ArgoCD` CR instead? Seems to be duplicated w/ `argocd/openshift-gitops` resource.

Reply:
Drop argocd template since sandbox has the same one.